examples/simple_tracking.cpp

```diff
@@ -112,6 +112,7 @@ int main(int argc, char *argv[]) {
   MultiTracker<Filter, 4> mtrk;    // state [x, v_x, y, v_y]
   CVModel cvm(5.0, 5.0);           // CV model with sigma_x = sigma_y = 5.0
   CartesianModel ctm(0.5, 0.5);              // Cartesian observation model
+  CartesianModel3D ctm(0.5, 0.5);              // Cartesian observation model
```

Comment:
Right, shouldn't the 3D model have 3 arguments anyway? The example is, by default, not compiled otherwise this would have crashed I guess.
I'd say remove that line and keep the example purely 2D. It would make more sense to copy this file and make a separate 3D example because a few other things would have to be changed for this as well. But for now, as I said, I'd just remove this line.